docs/editors/vim.md

```diff
@@ -185,8 +177,9 @@ command! -nargs=0 Format :call CocAction('format')
 " Use `:Fold` to fold current buffer
 command! -nargs=? Fold :call     CocAction('fold', <f-args>)
 
-" Add status line support, for integration with other plugin, checkout `:h coc-status`
-set statusline^=%{coc#status()}%{get(b:,'coc_current_function','')}
```

Comment:
I mention this below in the statusline section so I remove this here because if you _are_ enabling the statusline, you would want to use it that way more than likely. If you aren't enabling it, this doesn't do anything for you.